src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -202,7 +219,12 @@ public Task MessagesDeliveredAsync(IList<IBatchContainer> messages)
 
                 // clear cache and receiver
                 IEventHubQueueCache localCache = Interlocked.Exchange(ref cache, null);
-                EventHubReceiver localReceiver = Interlocked.Exchange(ref receiver, null);
+#if NETSTANDARD
+                PartitionReceiver
+#else
+                EventHubReceiver  
+#endif
```

Comment:
Not that we really have coding standards, but in general we don't use var unless the type is obvious, for code readability reasons.  I'd understand ignoring that in this case, but it's nice to know @galvesribeiro is not being lazy :)